vendor/src/github.com/bsphere/le_go/certs.go

```diff
@@ -0,0 +1,222 @@
+package le_go
+
+// Certs from https://logentries.com/doc/certificates
+var pemCerts = []byte(`
```

Comment:
@NathanMcCauley would it require upgrading docker to Go 1.7 to correctly read the system certs via the new `x509.SystemCertPool()` function?